tools/epub_reader.py

```diff
@@ -0,0 +1,169 @@
+import zipfile
+from bs4 import BeautifulSoup
+
+#Open epub
+#Convert html to markdown
+#Prints mardown to .json files
+
+
+def _init_():
+    tableOfContetns = []
+    listOfContetns = list()
```

Comment:
Python conventionally uses `snake_case` for variable naming. This one isn't as a big a deal but keep it in mind for the future. Additionally, watch out for typos.